packages/devtools_app/lib/src/service_registrations.dart

```diff
@@ -45,6 +45,17 @@ const RegisteredServiceDescription flutterVersion =
   icon: FlutterIcons.flutter,
 );
 
+/// Flutter memory service registered by Flutter Tools.
+///
+/// We call this service to get version information about the Flutter framework,
+/// the Flutter engine, and the Dart sdk.
+const RegisteredServiceDescription flutterMemory =
+    RegisteredServiceDescription._(
+  service: 'flutterMemoryInfo',
+  title: 'Flutter Memory Info',
+  icon: FlutterIcons.flutter,
```

Comment:
perhaps we should consult @raison00 for help with an icon

Reply:
Should be using memoryIcon which is the Octicons.package.  Changed.